src/ggrc/converters/handlers/acl.py

```diff
@@ -40,15 +40,17 @@ def _remove_people(self, people_list):
 
   def set_obj_attr(self):
     """Update current AC list with correct people values."""
-    if not self.value:
+    if not (self.value or self.set_empty):
```

Comment:
this function looks quiet complex, please refactor it please. 
for example move `(self.value or self.set_empty)` into variable